tests/bson/bson-timestamp_error-004.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+BSON BSON\Timestamp constructor requires positive 64-bit integers <= UINT32_MAX
```

Comment:
I believe I wanted to differentiate here because we do allow negative 32-bit integers (on 32-bit platforms), since we cast them to unsigned. The BSON timestamp type's two components are unsigned 32-bit integers. If we didn't cast 32-bit integers, we'd limit input to 2147483647 instead of 4294967295. I realize this is tricky and requires some documentation on PHP.net, but it seemed most flexible.

Anywho, only 64-bit integers are required to be positive, since we leave them as signed. How about:

```
requires 64-bit integers to be positive unsigned 32-bit integers
```